dependency-injection/2.0/_index.md

```diff
@@ -1,21 +1,24 @@
 ---
-title: "Jakarta Dependency Injection 2.0 (milestone)"
-date: 2020-05-18
+title: "Jakarta Dependency Injection 2.0"
+date: 2020-08-05
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
-* [Jakarta Dependency Injection 2.0 Release Record]()
+* [Jakarta Dependency Injection 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.cdi/releases/3.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Dependency Injection 2.0 Specification Document](./injection-spec-2.0-draft.pdf) (PDF)
-* [Jakarta Dependency Injection 2.0 Specification Document](./injection-spec-2.0-draft.html) (HTML)
+* [Jakarta Dependency Injection 2.0 Specification Document](jakarta-injection-spec-2.0.pdf) (PDF)
```

Comment:
As per the template, if directory is wombat, then spec file name should also have wombat

Based on that, I would expect the file name to be jakarta-injection-spec-2.0.pdf (same for html).
Or then use inject only for consistency with TCK filename, and Maven coordinates.

What do you think?